contracts/exchange-forwarder/contracts/src/MixinExchangeWrapper.sol

```diff
@@ -316,4 +316,40 @@ contract MixinExchangeWrapper is
             ));
         }
     }
+
+    /// @dev Checks whether an order's taker fee is effectively denominated in the maker asset.
+    ///      This is the case if they have the same ERC20Proxy asset data, or if the makerAssetData
+    ///      is the ERC20Bridge equivalent of the takerFeeAssetData.
+    /// @param takerFeeAssetData Byte array encoded for the takerFee asset proxy.
+    /// @param makerAssetData Byte array encoded for the maker asset proxy.
+    /// @return isPercentageFee Whether or not the taker fee asset matches the maker asset.
+    function _isPercentageFee(
```

Comment:
I might rename this something more generic, like `areAssetDataTransfersEqual`. I could see it being used elsewhere down the line.